core/modules/qdisp/Executive.cc

```diff
@@ -210,7 +214,7 @@ void Executive::markCompleted(int jobId, bool success) {
                 throw Bug(msg);
             }
         }
-        LOGS(_log, LOG_LVL_ERROR, "Executive: error executing jobId=" << jobId
+        LOGS(_log, LOG_LVL_ERROR, "Executive: error executing QId" << jobId
```

Comment:
Is `jobId` a job number (as opposed to query ID)? Should it be prefixed with `QId`?